### PR TITLE
Allow usage of pymumble without libopus when receive_sound is off

### PR DESCRIPTION
**Pull request by raek**
Proof of concept solution for issue #117.

### CODE REVIEW

File: pymumble_py3/mumble.py [hunk 1]
```diff
@@ -512,7 +518,7 @@ def sound_received(self, message):
 
             self.Log.debug("Audio frame : time:%f, last:%s, size:%i, type:%i, target:%i, pos:%i", time.time(), str(terminator), size, type, target, pos - 1)
 
-            if size > 0 and self.receive_sound:  # if audio must be treated
```
Comment (azlux) [inline pymumble_py3/mumble.py:515]: Why have you modified this line ?

**Review by azlux — COMMENTED** [review]
I like you commit. It perfect fit you need without disturbing normal use of the lib.
You can check the inline comment ?

File: pymumble_py3/mumble.py (hunk 1 above)
Comment (raek) [reply pymumble_py3/mumble.py:515]: After my change all sound-related methods are skipped in the dispatch_control_message() method. This means that sound_received() will not even be called, so the check on this line is redundant. self.receive_sound will always be true if we reach this line.

**Comment by raek** [comment]
@azlux : Do you want me to change anything or is this good to merge?

**Comment by azlux** [comment]
Srry, I was on another topic.
It's all good. I'll merge this PR.

Thank for your jobs on that.
Azlux

**Pull request merged**